### DISCUSSION TITLE
Home Control AS HC-SLM-1 OTA, now unresponsive

### DISCUSSION THREAD

**Discussion by DennisGreen80** [Koenkk/zigbee2mqtt#13527] [Devices]
I have a Wattle door lock with zigbee module Home Control AS HC-SLM-1
Since release 1.27.0 the configuration have failed, probably because it uses properties from a newer firmware than I had on the device.
That's why I decided to do a firmware upgrade OTA and it went fine.
Now it shows more options under exposes BUT it is now unresponsive.
I've tried restarting z2m, and also taking out the batteries from the door lock and reinsert them, didn't help.
Even worse, now when I hold down the zigbee module reset button, nothing happens.
Previously it starting blinking some green LEDs after 12sec. Now nothing.
I've also tried resetting the whole door lock, didn't help.
It feels like the device have been bricked.
Can anyone help?

**Comment by jb3112** [comment]
Did you solve the problem, i have same problem now. Even with Heimgaard hub
